### ISSUE TITLE
showing tooltip with node:mouseover event

### ISSUE THREAD

**Issue by agaurav** [q-m/d3.chart.sankey#1]
Is there way to use [d3.tip](https://github.com/Caged/d3-tip) with the `chart.on` if not then best way to add a `chart.tip` method simmilar to the one implemented here [line 102 and 156](http://jsfiddle.net/ramnathv/vtC75/2/) 

Also, great plugin (the only decent of sankey) and really liked the hierarchical highlighting of edges in product demo.

**Comment by wvengen** [comment]
Sorry for not responding earlier, somehow I've missed these notifications.
In theory this would be no problem, and I'd love d3.chart.sankey to support this. I'd say that you could subclass one of the d3.chart.sankey classes and setup d3.tip. Some untested rough idea:

``` js
var MySankeyTip = Sankey.extend("Sankey.Tip", {
  initialize: function() {
    var chart = this;
    chart.tip = // make the tip stuff work
  }
}); 
```

**Comment by agaurav** [comment]
Yeah sounds good, will submit a PR after trying it out. For now I am using the mouseover callbacks for showing the node or link details.

**Comment by wvengen** [comment]
Cool! I wonder if this should be a separate Sankey class, since there is no clear chain of derivation. It might be better to just create the chart, and then attach `d3.tip()` to it - so I'm curious where you'll end up. In any case, this would be useful to add to the documentation and/or as an example.
